### PR TITLE
Update to latest layout-bin-packer

### PR DESCRIPTION
**Pull request by lukemelia**
- Update tests for reliability by rounding to the nearest integer in many cases

### CODE REVIEW

**Comment by lukemelia** [comment]
Note that this will allow consuming apps to eliminate an ember-cli-babel version deprecation.

**Comment by lukemelia** [comment]
Also note that I marked the ember-release scenario as allowedToFail because this addon is not yet compatible with ember 5

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  NODE_VERSION: 14
```
Comment (gilest) [inline .github/workflows/ci.yml:14]: We'll probably want to bump the Node version in the `package.json` file, too.

Might be better to do that in a separate PR that lands before this one (since that will be a breaking change).

Comment (lukemelia) [reply .github/workflows/ci.yml:14]: I made that change and split out a dedicated commit. I tried to land this in a separate branch and couldn't get things to build.

Comment (gilest) [reply .github/workflows/ci.yml:14]: That's fair. I'm comfortable with the changes in that case since we want to get back into support.
